riff-raff.yaml

```diff
@@ -14,7 +14,7 @@ deployments:
     type: ami-cloudformation-parameter
     parameters:
       amiTags:
-        Recipe: ubuntu-xenial-java8
+        Recipe: ubuntu-bionic-capi
```

Comment:
bionic  capi  has  Java 8

Reply:
Description of recipe still refers to Java 11, which tripped me up for a moment. Looks good (though the scheduled bake failed this morning; I presume because this has yet to be merged in).